README.md

```diff
@@ -1,3 +1,10 @@
+
+<!--
+SPDX-FileCopyrightText: 2018, Marsiske Stefan 
+
+SPDX-License-Identifier: CC-BY-SA-4.0
```

Comment:
Our license recommendation for documentation is the  CC-BY-SA-4.0. I added the license text to the directory. Feel free to adopt it. :)

Reply:
good choice! i would have chosen this license